src/functionaltests/java/com/ericsson/ei/notifications/trigger/SubscriptionNotificationSteps.java

```diff
@@ -208,7 +209,7 @@ public void missed_notification_db_should_contain_x_objects(int maxObjectsInDB)
         int minWaitTime = 5;
         int maxWaittime = 20;
 
-        String condition = "{}";
+        MongoCondition condition = MongoCondition.emptyCondition();
```

Comment:
condition can be final in almost all the places

Reply:
Fixed in 701527f